lib/order.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'customer'
+require 'csv'
+require 'pry'
+
+class Order
+  attr_reader :id
+  attr_accessor :customer, :products, :fulfillment_status
+
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id.to_i
+    @products = products
+    @customer = customer
+    if [:pending, :paid, :processing, :shipped, :complete].include?(fulfillment_status)
+      @fulfillment_status = fulfillment_status
+    else
+      raise ArgumentError
+    end
+  end
+
+  def total
+    if @products == {} or @products == 0
+      return 0
```

Comment:
If you didn't have this `if`/`else` in this method, would the implementation of finding `total` not be `0` if `@products` were an empty hash or `0`?